pkg/harvester/edit/harvesterhci.io.addon/harvester-seeder.vue

```diff
@@ -0,0 +1,58 @@
+<script>
+import Tabbed from '@shell/components/Tabbed';
+import Tab from '@shell/components/Tabbed/Tab';
+import { RadioGroup } from '@components/Form/Radio';
+
+import CreateEditView from '@shell/mixins/create-edit-view';
+
+export default {
+  name:       'EditAddonPCI',
+  components: {
+    Tabbed,
+    Tab,
+    RadioGroup,
+  },
+
+  mixins: [CreateEditView],
+
+  props: {
+    value: {
+      type:     Object,
+      required: true,
+    },
+
+    mode: {
+      type:     String,
+      required: true
+    },
+  },
+};
+</script>
+
+<template>
+  <Tabbed :side-tabs="true">
+    <Tab
+      name="basic"
+      :label="t('harvester.addons.vmImport.titles.basic')"
+      :weight="99"
+    >
+      <RadioGroup
+        v-model="value.spec.enabled"
+        class="mb-20"
+        name="model"
+        :mode="mode"
+        :options="[true,false]"
+        :labels="[t('generic.enabled'), t('generic.disabled')]"
+      />
+    </Tab>
+  </Tabbed>
+</template>
+
+<style lang="scss" scoped>
+  ::v-deep .radio-group {
+    display: flex;
+    .radio-container {
+      margin-right: 30px;
+    }
+  }
+</style>
```

Comment:
We should have a component that is rendered by default if there is no other custom form.        
An issue can be created to track this enhancement.